.github/workflows/ci.yml

```diff
@@ -57,7 +57,10 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
+          ssh-key: ${{ secrets.GH_BOT_TOKEN }}
```

Comment:
I can't really inspect the secret, but my gut tells me its not an SSH key of our bot.